lib/errors.js

```diff
@@ -82,6 +82,3 @@ errors.LusterPortError = LusterError.create('LusterPortError',
         CAN_NOT_UNLINK_UNIX_SOCKET :
             'Can not unlink unix socket "%socketPath%"'
     });
-
-module.exports = errors;
-
```

Comment:
It's not specified by linters rules, but, i think, it's common style in all the nodules. What the reason to break it?